src/components/molecules/tabs/request/response/HeadersDisplayTable.tsx

```diff
@@ -0,0 +1,59 @@
+import { AccordionGroup, Accordion, AccordionSummary, AccordionDetails, Table } from '@mui/joy';
+import { SPHeaders } from '../../../../../types/application-data/application-data';
+
+export function HeadersDisplayTable({
+	headers,
+	label,
+}: {
```

Comment:
I find stuff more readable if the typing for the props is pulled out into an interface.